src/components/ui/Background/Background.styled.js

```diff
@@ -0,0 +1,14 @@
+import styled from 'styled-components';
+
+const Background = styled.div`
```

Comment:
What's the purpose of the background element? Is it only displayed when the dropdown is active?